### PR TITLE
3rd and 4th task of Exceptions

### PR DESCRIPTION
**Pull request by IvayloTsvetkov**
Creating a list class with add, remove and printAllElements methods that throws exceptions when list is full or empty

### CODE REVIEW

**Comment by IvayloTsvetkov** [comment]
Not sure if I did anything right.

**Comment by mgenov** [comment]
Doesn't look so as I'm still seeing the code from previous PR.

**Comment by IvayloTsvetkov** [comment]
I hope it's alright now.

File: Exceptions_3/src/com/company/MyList.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.company;
+
+/**
+ * Created by clouway on 13.07.17.
+ */
+public class MyList {
+    private Object[] arr;
+    private int size, pos;
```
Comment (mgenov) [inline Exceptions_3/src/com/company/MyList.java]: You don't need size as arrays in java are having such properties.

File: Exceptions_3/src/com/company/MyList.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    MyList(int size) {
+        this.size = size;
+        arr = new Object[size];
+        pos = 0;
+    }
+
+    public void add(Object obj) {
+        if(pos == size) throw new ListFullException(size);
```
Comment (mgenov) [inline Exceptions_3/src/com/company/MyList.java]: Add {}

File: Exceptions_3/src/com/company/MyList.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        arr[pos++] = obj;
+    }
+
+    public void remove() {
+        if(pos == 0) throw new ListEmptyException();
```
Comment (mgenov) [inline Exceptions_3/src/com/company/MyList.java]: Add {}

**Comment by IvayloTsvetkov** [comment]
Since I'm doing my projects in separate folders I had to change all ignored files/directories in .gitignore to start with **/. I'll do it right with I/O.

**Comment by IvayloTsvetkov** [comment]
I made the changes.

**Comment by mgenov** [comment]
I'm not able to see them.

**Comment by IvayloTsvetkov** [comment]
Done. Forgot to push the changes.

**Pull request merged**